pkg/hubble/parser/options/options.go

```diff
@@ -53,9 +53,10 @@ func Redact(logger logrus.FieldLogger, httpQuery, httpUserInfo, kafkaApiKey bool
 			Deny:  headerSliceToMap(denyHeaders),
 		}
 		if logger != nil {
-			logger.WithField(
-				"options",
-				fmt.Sprintf("%+v", opt)).Info("configured Hubble with redact options")
+			logger.Info(
+				"configured Hubble with redact options",
+				logfields.Options, opt,
```

Comment:
out of curiosity: do we get the same format output as `fmt.Sprintf("%+v", opt)` here?

Reply:
We do! I ended up writing a test after your comment to confirm it. (see `pkg/hubble/parser/options/options_test.go`)

I used this format with "main" and then copied the output from the buffer and used that as the expected result from the logger output of the test.
```
	formatter := &logrus.TextFormatter{
		DisableColors:    true,
		DisableTimestamp: true,
	}
	logger := logrus.New()
	logger.SetOutput(&buf)
	logger.SetFormatter(formatter)
	logger.SetLevel(logrus.DebugLevel)
```